multiversx_sdk_cli/cli_shared.py

```diff
@@ -172,6 +175,28 @@ def prepare_nonce_in_args(args: Any):
         args.nonce = account.nonce
 
 
+def prepare_chain_id_in_args(args: Any):
+    if not args.chain and not args.proxy:
+        raise ArgumentsNotProvidedError("chain or proxy should be provided")
```

Comment:
`chain ID cannot be inferred / decided: --chain or --proxy should be provided` (alternative).

Reply:
Replaced.